### PR TITLE
[4.4] Contact VCF: Properly assign value to array

### PR DESCRIPTION
**Pull request by Hackwar**
### Summary of Changes
While going through the code with phpstan, this showed up as error. Instead of assigning the string to a new entry in the array, we are also trying to read from this array by trying to concatenate a string... I'm struggling to explain it, but if you look at the code, you see the problem. Right now this does not throw an error in PHP, but it is just a matter of time when PHP becomes strict enough to complain about this.

### Testing Instructions
Codereview

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a1b297a63bbd0f27833af185c07016d27010d94f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43353">issues.joomla.org/tracker/joomla-cms/43353</a>.</sub>

File: components/com_contact/src/View/Contact/VcfView.php [hunk 1]
```diff
@@ -89,20 +89,20 @@ public function display($tpl = null)
         Factory::getApplication()->setHeader('Content-disposition', 'attachment; filename="' . $card_name . '.vcf"', true);
 
         $vcard = [];
```
Comment (Quy) [inline components/com_contact/src/View/Contact/VcfView.php]:
```suggestion
        $vcard   = [];
```

**Review by HLeithner — APPROVED** [review]
code review

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 59e734646b977deecb2bb2a0e730682e392ba6c6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43353">issues.joomla.org/tracker/joomla-cms/43353</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43353">issues.joomla.org/tracker/joomla-cms/43353</a>.</sub>

**Comment by MacJoom** [comment]
Thank you!

**Pull request merged**